weave-js/src/components/PagePanelComponents/Home/Browse3/pages/MonitorsPage/MonitorPage.tsx

```diff
@@ -0,0 +1,275 @@
+                    sx={{...typographyStyle}}
+                    className="font-semibold">
+                    Monitor details
+                  </Typography>
+                </Box>
+                <SimpleKeyValueTable
+                  data={{
+                    Description: monitorVersions[0].val['description'],
+                    Active: (
+                      <BooleanIcon value={monitorVersions[0].val['active']} />
+                    ),
+                    'Monitored Ops': (
+                      <Box className="flex gap-2">
+                        {monitorVersions[0].val['op_names'].map(
+                          (opRefUri: string) => (
+                            <SmallOpVersionsRef
+                              key={opRefUri}
+                              objRef={parseRef(opRefUri) as WeaveObjectRef}
+                            />
+                          )
+                        )}
+                      </Box>
+                    ),
+                    'Additional filters': (
+                      <Box className="flex gap-2">
+                        {filterItems.map(item => (
+                          <FilterTagItem
+                            key={item.id}
+                            item={item}
+                            disableRemove
+                          />
+                        ))}
+                      </Box>
+                    ),
+                    'Sampling rate': `${
+                      monitorVersions[0].val['sampling_rate'] * 100
+                    }%`,
+                    Scorers: (
+                      <Box className="flex gap-2">
+                        {monitorVersions[0].val['scorers'].map(
+                          (scorerRefUri: string) => (
+                            <SmallRef
+                              objRef={parseRef(scorerRefUri)}
+                              key={scorerRefUri}
+                            />
+                          )
+                        )}
+                      </Box>
+                    ),
+                  }}
+                />
+              </Tailwind>
+            </ScrollableTabContent>
+          ),
+        },
+        {
+          label: 'Calls',
+          content: (
+            <Tailwind>
+              <Box className="flex flex-col gap-16 p-16">
+                <Box className="flex justify-between">
+                  <span>
+                    {callCountResult && (
+                      <>
+                        {callCountResult.count}{' '}
+                        {maybePluralizeWord(callCountResult.count, 'call')}
+                      </>
+                    )}
+                  </span>
+                  <Button
+                    variant={ButtonVariants.Secondary}
+                    icon={IconNames.Table}
+                    onClick={onGoToTableClick}>
+                    Go to table
+                  </Button>
+                </Box>
+                <CallsTable
```

Comment:
![Screenshot 2025-05-13 at 16 07 21](https://github.com/user-attachments/assets/bceade5c-625c-4455-98ff-58e4c84f8659)

* Would be nice if the attributes were hidden by default like they are in the main table
* Would be nice to take up 100% of allowed height

Reply:
I'll address this in a follow-up PR.